http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSAlgebra.scala

```diff
@@ -35,25 +33,6 @@ final case class UpdateAKSAppParams(appId: AppId,
 final case class DeleteAKSAppParams(
   appName: AppName,
   workspaceId: WorkspaceId,
-  landingZoneResourcesOpt: LandingZoneResources,
-  cloudContext: AzureCloudContext
+  cloudContext: AzureCloudContext,
+  billingProfileId: BillingProfileId
 )
-
-/** Enumerates the possible identity modes for an AKS app. */
-sealed trait IdentityType
```

Comment:
I thought I removed this in a prior PR… it is obsolete, we are using WorkloadIdentity across the board now.